extension/src/paymentHandler/payment-utils.js

```diff
@@ -66,6 +67,16 @@ function createSetCustomFieldAction(name, response) {
   }
 }
 
+function createSetMethodInfoMethodAction(paymentMethod) {
+  const paymentMethodsToNames = getPaymentMethodsToNamesConfig()
+  const paymentMethodName =
+    paymentMethodsToNames[paymentMethod] || paymentMethod
```

Comment:
better to escape the `paymentMethod` to avoid any broken query to ctp platform.

Reply:
I don't think we need to escape here. According to the docs there's no restrictions: https://docs.commercetools.com/api/projects/payments#paymentmethodinfo